_docs/_latest/setup/manual.md

```diff
@@ -88,9 +87,9 @@ by their tags:
     git checkout tags/<tag_number>
     ``` 
     
-1. Follow the setup instructions in the
-[startup-script](https://github.com/GoogleCloudPlatform/forseti-security/blob/stable/deployment-templates/compute-engine/server/forseti-instance-server.py#L109) 
-to install Forseti server and all the necessary components like Fluentd and Cloud SQL Proxy.
+1. To install Forseti server and necessary components like Fluentd
+and Cloud SQL Proxy, follow the setup instructions in the
+[startup-script](https://github.com/GoogleCloudPlatform/forseti-security/blob/stable/deployment-templates/compute-engine/server/forseti-instance-server.py#L109).
```

Comment:
@joecheuk and @blueandgold can you help Tabitha here and search _docs/_latest for any github code links that are NOT pointed at /stable/ ?

Reply:
Resolved in https://github.com/GoogleCloudPlatform/forseti-security/pull/1690/commits/672c249ff377bd060143cd6b147ce2e787bdb289